index.js

```diff
@@ -2179,6 +2184,13 @@
     };
   }
 
+  //  see :: (String, { name :: String, url :: String }) -> String
+  function see(label, record) {
+    return record.url &&
```

Comment:
I believe that we can release a `patch` to sanctuary-def in order to combat this issue, and allow users to enjoy using their outdated fluture or other sanctuary-type-classes based library with sanctuary-def without trailing `undefined`s in their error messages. ;)